ts/src/pro/bitstamp.ts

```diff
@@ -33,6 +33,7 @@ export default class bitstamp extends bitstampRest {
                 'watchOrderBook': {
                     'snapshotDelay': 6,
                     'maxRetries': 3,
+                    'initialFetchOrderBookMaxRetries': 3,
```

Comment:
@ttodua I find this name a bit weird, if the orderbook becomes invalid over time we might need to re-fetch it so "initially" is not quite right. Why not something like `maxSnapshotRetries`, we're already inside the `watchOrderBook` option so it should be clear even without `orderbook` in the name